### DISCUSSION TITLE
Statistics of orientations

### DISCUSSION THREAD

**Discussion by hchaieb1234** [mtex-toolbox/mtex#1336] [Ask Anything]
Hello, could you please help me in identifiying the pourcentage of grains that have an orientation that is close to a specific orientation (basal for example) in numbers, startig from a set of EBSD data? For example I would like to know that basal orientation represents x% of the measured data. Thank you.

**Comment by kilir** [comment]
Hi,
if you want number fraction of grains, can do something like:
```
o = grains('xy').meanOrientation;
cond = angle(o,basal_o)<15*degree
sum(cond)/length(cond)
```
where `basal_o` is something of your interest you need to define.
Cheers,
Rüdiger
